303-quarkus-vertx-sql/pom.xml

```diff
@@ -51,15 +63,194 @@
+                                <tmpfs>/var/lib/mysql</tmpfs>
+                                <wait>
+                                    <log>.*MySQL Community Server.*</log>
+                                    <time>20000</time>
+                                    <!-- Kill the container, if it doesn't stop before this given time
+                                     duration since a graceful stop was issued -->
+                                    <kill>300000</kill>
+                                </wait>
+                            </run>
+                        </image>
+                        <image>
+                            <name>${db2.image}</name>
+                            <alias>db2</alias>
+                            <run>
+                                <privileged>true</privileged>
+                                <ports>
+                                    <port>db2.port:50000</port>
+                                </ports>
+                                <env>
+                                    <DB2INSTANCE>test</DB2INSTANCE>
+                                    <DB2INST1_PASSWORD>${db.password}</DB2INST1_PASSWORD>
+                                    <DBNAME>${db.name}</DBNAME>
+                                    <LICENSE>accept</LICENSE>
+                                    <!-- These help the DB2 container start faster -->
+                                    <AUTOCONFIG>false</AUTOCONFIG>
+                                    <ARCHIVE_LOGS>false</ARCHIVE_LOGS>
+                                    <PERSISTENT_HOME>false</PERSISTENT_HOME>
+                                </env>
+                                <log>
+                                    <prefix>DB2:</prefix>
+                                    <date>default</date>
+                                    <color>cyan</color>
+                                </log>
+                                <wait>
+                                    <log>.*Setup has completed.*</log>
+                                    <!-- Unfortunately booting DB2 is slow, needs to set a generous (15m) timeout;
+                                     it generally starts in 2-3 minutes, but it's been occasionally slightly above 10m -->
+                                    <time>900000</time>
+                                    <!-- Kill the container, if it doesn't stop before this given time
+                                     duration since a graceful stop was issued -->
+                                    <kill>300000</kill>
+                                </wait>
+                            </run>
+                        </image>
+                    </images>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>docker-start</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>stop</goal>
```

Comment:
This stop is not required and could give problem on parallel test runs

Reply:
Removed